packages/frinx-uniresource-ui/src/pages/resource-types-page/create-resource-type-modal.tsx

```
@@ -12,35 +13,74 @@ import {
   ModalFooter,
   ModalHeader,
   ModalOverlay,
-  Spacer,
 } from '@chakra-ui/react';
 import * as yup from 'yup';
-import { useFormik } from 'formik';
+import { FormikValues, useFormik } from 'formik';
+import { unwrap } from '@frinx/shared';
+import ExpectedProperties, { ExpectedProperty } from '../../components/expected-properties-form';
 
 type Props = {
   isOpen: boolean;
   onClose: () => void;
   onCreate: (resourceTypeName: string) => void;
 };
 
+type FormValues = {
+  resourceTypeName: string;
+  resourceTypeProperties?: ExpectedProperty[];
+};
+
+// eslint-disable-next-line func-names
```

You don't really have to suppress this error like this, just name the function whatever.